practice_project/main.py

```diff
@@ -1,26 +1,16 @@
 import unittest
 import math
-def is_prime(n):
-    if n <= 1:
+
+def is_prime(number):
+    if number <= 1:
         return False
-    for i in range(2, int(math.sqrt(n)) + 1):
-        if n % i == 0:
+    for i in range(2, int(math.sqrt(number)) + 1):
+        if number % i == 0:
             return False
     return True
```

Comment:
you should separate any source files from test files, it's industry standard to not mix the two concepts up in the same files